test/test_utils/__init__.py

```diff
@@ -1690,34 +1445,50 @@ def get_neuron_sdk_version_from_tag(image_uri):
     return neuron_sdk_version
 
 
-def get_neuron_framework_and_version_from_tag(image_uri):
+def get_neuron_release_manifest(sdk_version):
     """
-    Return the framework version and expected framework version for the neuron tag from the image tag.
+    Returns a dictionary which maps a package name (e.g. "tensorflow-neuronx") to all the version numbers available in and SDK release
 
-    :param image_uri: ECR image URI
-    :return: framework version, expected framework version from neuron sdk version
+    :param sdk_version: Neuron SDK version
+    :return: { "tensorflow_neuronx": set(["15.5.1.1.1.1.1", "2.10.1.1.1.1.1", ...]), ... }
     """
-    tested_framework, tag_framework_version = get_framework_and_version_from_tag(image_uri)
-    neuron_sdk_version = get_neuron_sdk_version_from_tag(image_uri)
+    NEURON_RELEASE_MANIFEST_URL = "https://raw.githubusercontent.com/aws-neuron/aws-neuron-sdk/master/src/helperscripts/n2-manifest.json"
 
-    if neuron_sdk_version is None:
-        return tag_framework_version, None
+    manifest = requests.get(NEURON_RELEASE_MANIFEST_URL)
```

Comment:
do we want to add any safeguards if the manifest is empty for any reason?

Reply:
It now asserts on it being empty, the test can't continue properly without content being present